.github/workflows/deploy.yaml

```
@@ -0,0 +1,91 @@
+          echo $TASK_DEFINITION | jq \
+            --arg IMAGE "${{ vars.ECR_REPOSITORY_URI }}:latest" \
+            '.taskDefinition | .containerDefinitions[0].image = $IMAGE | del(.taskDefinitionArn, .revision, .status, .requiresAttributes, .compatibilities, .registeredAt, .registeredBy)'
+          )
+
+          # 새로운 태스크 정의 등록
+          NEW_TASK_DEF_ARN=$(
+            aws ecs register-task-definition \
+              --cli-input-json "$NEW_TASK_DEFINITION" | \
+            jq -r '.taskDefinition.taskDefinitionArn'
+          )
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)